src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/TaskAwaiter.cs

```diff
@@ -78,7 +78,7 @@ internal TaskAwaiter(Task task)
         /// <param name="continuation">The action to invoke when the await operation completes.</param>
         /// <exception cref="System.ArgumentNullException">The <paramref name="continuation"/> argument is null (Nothing in Visual Basic).</exception>
         /// <exception cref="System.InvalidOperationException">The awaiter was not properly initialized.</exception>
-        /// <remarks>This method is intended for compiler user rather than use directly in code.</remarks>
+        /// <remarks>This method is intended for compiler use rather than use directly in code.</remarks>
```

Comment:
This only changed this one comment.  The same typo appears in other places, e.g. in the very next function on L91.